ckan-docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM openknowledge/ckan-dev:2.8
+
+MAINTAINER Your Name Here <you@example.com>
+
+# Install any extensions needed by your CKAN instance
+# (Make sure to add the plugins to CKAN__PLUGINS in the .env file)
+# For instance:
+RUN pip install -e git+https://github.com/ckan/ckanext-spatial#egg=ckanext-spatial && \
+    pip install -r https://raw.githubusercontent.com/ckan/ckanext-spatial/master/pip-requirements.txt && \
+    pip install -e git+https://github.com/ckan/ckanext-harvest.git#egg=ckanext-harvest && \
+    pip install -r https://raw.githubusercontent.com/ckan/ckanext-harvest/master/pip-requirements.txt 
```

Comment:
Let's install from a single requirements.txt. Having multiple `pip install`s could end up with hidden conflicting dependencies.

Reply:
Trying to figure out how to move these to the requirements file, they do not install correctly just by placing them there for the moment

Reply:
I don't think there's a good way. I've been copy/pasting the contents of each one into a single file.

Reply:
Is that what you think we should do here?

Reply:
Yeah, it should be moved to a single requirements file, unless there is a better practice. It should really be in setup.py, but CKAN community doesn't really do that.